bitnami/common/templates/_secrets.tpl

```diff
@@ -91,7 +91,7 @@ The order in which this function returns a secret password:
 
 {{- $password := "" }}
 {{- $subchart := "" }}
-{{- $failOnNew := default true .failOnNew }}
+{{- $failOnNew := default "true" .failOnNew }}
```

Comment:
I just saw that you forgot to update "Usage" information (see line 80) defining:
```diff
-  - failOnNew - Boolean - Optional - Default to true. If set to false, skip errors adding new keys to existing secrets.
+  - failOnNew - String - Optional - Default to "true". If set to "false", skip errors adding new keys to existing secrets.
```